cmake/GoogleCloudCppFeatures.cmake

```diff
@@ -225,6 +225,11 @@ function (google_cloud_cpp_enable_features)
             if (NOT ("storage" IN_LIST GOOGLE_CLOUD_CPP_ENABLE))
                 add_subdirectory(google/cloud/storage)
             endif ()
+        elseif ("${feature}" STREQUAL "http-transcoding")
```

Comment:
In this the final form of this feature?  If we have any doubts (e.g. how to support this in other services beyond spanner, where we need globs to work) then I would call it "experimental-http-transcoding".